2_process.R

```diff
@@ -16,6 +17,39 @@ p2_targets_list <- list(
                        sites = p1_drb_temp_sites_sf)
   ),
   
+  tar_target(p2_buffered_nhd_reaches,
+              st_buffer(p1_nhd_reaches, dist = 250)
+  ),
+
+  tar_target(p2_buffered_nhd_reaches_along_nhm,
+              st_buffer(p1_nhd_reaches_along_NHM, dist = 250)
+  ),
+
+  # Catchment -- Will be updated to upstream catchments
+  tar_target(p2_depth_to_bedrock_catchments,
+             raster_in_polygon_weighted_mean(raster = '1_fetch/in/Shangguan_dtb_cm_250m_clip 2/w001001.adf',
```

Comment:
Can you add a comment that indicates how a user should access/download this file? How big is it? `'1_fetch/in/Shangguan_dtb_cm_250m_clip 2/w001001.adf'`

Also, do you think we should add a file format target for `'1_fetch/in/Shangguan_dtb_cm_250m_clip 2/w001001.adf'` in `1_fetch.R`?

Reply:
Thanks, Margaux for adding these comments! I also see that you had previously given the data provenance and instructions that it should be manually placed in 1_fetch/in. I'm really sorry for missing that!

Related to my comment above, I think it's fine to leave the url defined in `_targets.R`, but I'd also recommend we create a new target associated with this dataset in `1_fetch.R`, e.g.

```r
  # Track depth to bedrock raster dataset in 1_fetch/in
  tar_target(
    p1_depth_to_bedrock_tif,
    Shangguan_dtb_cm_250m_clip_path,
    format = "file"
  )
```

If we have a file target that corresponds to this file `targets` will track any changes in that file and update downstream dependencies accordingly. If we plan to eventually download the dataset directly, this target can also serve as a placeholder for that fetch step.